entities/vectorindex/hnsw/config.go

```diff
@@ -209,7 +223,7 @@ func (u *UserConfig) validate() error {
 	return nil
 }
 
-// Tries to parse the int value from the map, if it overflows math.MaxInt64, it
+// Tries to parse the int value from the map, if it overflows Int64, it
```

Comment:
I would leave comment about `math.MaxInt64` as it's more verbose

Reply:
The full comment was:
// Tries to parse the int value from the map, if it overflows math.MaxInt64, it
// uses math.MaxInt64 instead. This is to protect from rounding errors from
// json marshalling where the type may be assumed as float64
I changed this, because in the code it tries Int64 and then uses math.MaxInt64 instead when it overflows.

Reply:
On second thought I decided to remove this comment and the associated code entirely, because it relies on the false assumption that json marshalling might interpret the number as float64. The funtion used: `typed.Int64()` does not do that. So that bit of code will only ever confuse user, who added a number that is too large.

Reply:
And on third thought. This should likely be done in a seperate PR. Reverted it back.